sw/airborne/modules/computer_vision/lib/vision/image.c

```diff
@@ -596,7 +596,12 @@ void image_show_flow(struct image_t *img, struct flow_t *vectors, uint16_t point
  * @param[in] *from The point to draw from
  * @param[in] *to The point to draw to
  */
-void image_draw_line(struct image_t *img, struct point_t *from, struct point_t *to)
+void image_draw_line(struct image_t *img, struct point_t *from, struct point_t *to) {
+  static uint8_t color[4] = {255, 255, 255, 255};
+  image_draw_line_color(img, from, to, color);
+}
+
+void image_draw_line_color(struct image_t *img, struct point_t *from, struct point_t *to, uint8_t *color)
```

Comment:
Please add doxygen documentation for this function, especially what you should pass in color...